src/components/UserInfo.vue

```diff
@@ -1,24 +1,42 @@
+<!--suppress XmlInvalidId -->
 <template>
-  <div id="user-info">
-    <v-alert :value="!!successMessage" type="success" dismissible>{{ successMessage }}</v-alert>
-    <v-alert :value="!!errorMessage" type="error" dismissible>{{ errorMessage }}</v-alert>
+  <div id="user-info" v-if="dataReady">
+    <v-alert v-model="successState" type="success" dismissible>{{ successMessage }}</v-alert>
+    <v-alert v-model="errorState" type="error" dismissible>{{ errorMessage }}</v-alert>
```

Comment:
Using `successState` and `errorState` fixes a bug where the messages can only be shown once. That is, if there is an error, and you dismiss the message, subsequent errors won't re-show the message. Same for success: if you update the user once, dismiss the success message, and then update again, the message won't reappear.